src/Unit/AbstractUnit.php

```diff
@@ -58,59 +68,82 @@ public function getInstanceName(): ?string
     }
 
     /**
+     * @return string
+     */
+    abstract protected function getUnitSuffix(): string;
+
+    /**
+     * @param array $commands
+     *
+     * @return CommandInterface
+     */
+    public function execute(...$commands): CommandInterface
+    {
+        $commands[] = implode(
+            '.',
+            [
+                $this->name,
+                $this->getUnitSuffix(),
```

Comment:
Because the constant is not a requirement.
I wanted everyone who extended the AbstractUnit to be forced to implement a method that I can use.
And since it is not possible to `abstract` constants and it is not possible to put protected methods in an interface, this was the way to go.